version/version.cc

```diff
@@ -35,7 +35,7 @@ const bool Version::withDebugSymbols = true;
 const bool Version::withDebugSymbols = false;
 #endif
 
-const string Version::version = "0.4"; // 0.01 alpha
+const string Version::version = "0.5"; // 0.01 alpha
```

Comment:
I don't remember where we ended up on whether to bump the version number or not. Did we say we'd do it or put it off?

Reply:
If we said we'd do it, I'm surprised to only see it changing here.

Reply:
It looks like the command I was using to search for it wasn't finding hidden files, because there were a bunch of places in buildkite scripts. I'll go through more.